src/utils/components/NodeSelectorModal/components/LabelRow.tsx

```diff
@@ -13,9 +13,19 @@ type LabelRowProps = {
   onDelete: (id: any) => void;
 };
 
-const LabelRow: React.FC<LabelRowProps> = ({ label, onChange, onDelete }) => {
+const LabelRow: FC<LabelRowProps> = ({ label, onChange, onDelete }) => {
   const { t } = useKubevirtTranslation();
   const { id, key, value } = label;
+
+  const handlePasteLabelKey = (event): void => {
+    event.preventDefault();
+    const text = event.clipboardData.getData('text');
+    const symbols = text.split('=');
+
+    if (symbols.length > 1) onChange({ ...label, key: symbols[0], value: symbols[1] });
```

Comment:
can add an empty return and remove the else

Reply:
Fixed!